server/game/gameSystems/GameSystemLibrary.ts

```diff
@@ -314,17 +306,9 @@ export function discardCardsFromOwnHand<TContext extends AbilityContext = Abilit
 export function discardCardsFromOpponentsHand<TContext extends AbilityContext = AbilityContext>(propertyFactory: PropsFactory<IDiscardCardsFromHandProperties, TContext>): DiscardCardsFromHandSystem<TContext> {
```

Comment:
Just noticed this - since we're forcing the value of the `discardingRelativePlayer` property we should use an `Omit<>` type to remove that from the `propertyFactory` type completely so that the caller can't try to specify it. Take a look at `playCardFromHand()` and `playCardFromOutOfPlay()` in this file for example of how to use the `Omit<>`

Reply:
Thanks for this, I was wondering how I could make it non-optional and not require them to pass it on at the top level.   Added this.